src/assets/body/tail/tail.asset.ts

```diff
@@ -0,0 +1,129 @@
+DefineBodypart({
+	name: 'Tail',
+	bodypart: 'tails',
+	allowRandomizerUsage: false,
+	graphics: 'graphics.json',
+	colorization: {
+		tail: {
+			name: 'Tail',
+			default: '#666666',
+		},
+		stripes: {
+			name: 'Stripes (not for all types)',
+			default: '#020202',
+		},
+		tip: {
+			name: 'Tip (not for all types)',
+			default: '#FFFFFF',
+		},
+	},
+	preview: null,
+	attributes: {
+		provides: [
+			'Tail',
+		],
+	},
+	modules: {
+		tailType: {
+			type: 'typed',
+			name: 'Tail Type',
+			variants: [
+				{
+					id: 'bunny',
+					name: 'Bunny',
+				},
+				{
+					id: 'cat',
+					name: 'Cat',
+				},
+				{
+					id: 'thin',
+					name: 'Thin',
+				},
+				{
+					id: 'normal',
+					name: 'Normal',
+					default: true,
+				},
+				{
+					id: 'thick',
+					name: 'Thick',
+				},
+			],
+		},
+		withStripes: {
+			type: 'typed',
+			name: 'With Stripes',
```

Comment:
Not all these options work with all tail types. This could be solved by splitting the asset into multiple assets (mainly looking at the bunny tail, which I think would make sense as a separate asset with far fewer modules), or by using state flags.
This applies to all the modules (other than "Tail Type")